pkg/library/flatten/flatten.go

```diff
@@ -29,12 +30,21 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	// DWTSupportedNamespacesAnnotation defines additional namespaces from which a DevWorkspace can import a DevWorkspaceTemplate.
+	// Options are:
+	// - '*': allow importing by all DevWorkspaces on the cluster
+	// - 'namespaceA,namespaceB,namespaceC': Allow importing by DevWorkspaces in list of specific namespaces
```

Comment:
I don't think that anybody care about such a case, but for clarity, even when you specify `namespaceA`, the current namespace should still work as well.

Reply:
Clarified. Alternatively, we could change the annotation to `allow-all-namespaces: ['true'|'false']` if that's simpler.